frontend/src/helpers/UserAuthApi.js

```diff
@@ -15,7 +17,7 @@ const UserAuthApi = {
       })
     };
 
-    return fetch('http://localhost:8000/api/register', options);
+    return fetch(`${ServerPath.api}/register`, options);
```

Comment:
just an example but this should now look like this when we apply the change above:
```suggestion
    return fetch(`${process.env.REACT_APP_API}/register`, options);
```